tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot

```diff
@@ -6,20 +6,65 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 Default Tags
 
 *** Test Cases ***
-Display HTML
+Get Login Page
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/authentication
+    Should contain  ${output}  <title>VCH Admin</title>
+
+While Logged Out Fail To Display HTML
     ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}
+    Should not contain  ${output}  <title>VIC: ${vch-name}</title>
+
+While Logged Out Fail To Get Portlayer Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/port-layer.log
+    Should Not Contain  ${output}  Launching portlayer server
```

Comment:
This one and the ones below it are more specific and less dangerous, but it would still be good to add what you actually expect the user to see.